apps/emqx_authz/src/emqx_authz.appup.src

```diff
@@ -1,8 +1,13 @@
+%% -*- mode: erlang -*-
 %% Unless you know what you are doing, DO NOT edit manually!!
 {VSN,
-  [{"0.1.1",
-    [{load_module,emqx_authn_utils,brutal_purge,soft_purge,[]}]},
+  [{"0.1.0",[{load_module,emqx_authz_utils,brutal_purge,soft_purge,[]}]},
+   {"0.1.1",
+    [{load_module,emqx_authz_utils,brutal_purge,soft_purge,[]},
+     {load_module,emqx_authn_utils,brutal_purge,soft_purge,[]}]},
```

Comment:
0.1.1 is not released yet.

Reply:
But it was wrongly bumped in the app src file, so the script filled in all the gaps, assuming the internal applications follow semver. It's too time consuming to fight this.